tsdb/db_test.go

```diff
@@ -58,16 +58,15 @@ func TestMain(m *testing.M) {
 }
 
 func openTestDB(t testing.TB, opts *Options, rngs []int64) (db *DB) {
-	tmpdir, err := ioutil.TempDir("", "test")
-	require.NoError(t, err)
+	tmpdir := t.TempDir()
 
 	if len(rngs) == 0 {
-		db, err = Open(tmpdir, nil, nil, opts, nil)
+		db, _ = Open(tmpdir, nil, nil, opts, nil)
 	} else {
 		opts, rngs = validateOpts(opts, rngs)
-		db, err = open(tmpdir, nil, nil, opts, rngs, nil)
+		db, _ = open(tmpdir, nil, nil, opts, rngs, nil)
 	}
-	require.NoError(t, err)
+	require.NoError(t, nil)
```

Comment:
We should not ignore the error here. You can have `var err error` before the `if` and check the error.